src/schema/rules/tabular_data/motion.yaml

```diff
@@ -0,0 +1,26 @@
+---
+motionChannels:
+  selectors:
+    - datatype == "motion"
+    - suffix == "channels"
+    - extension == ".tsv"
+  initial_columns:
+    - name__channels
+    - component__channels
+    - type__motion_channels
+    - tracked_point__channels
+    - units__motion
+  columns:
+    name__channels: required
+    component__channels: required
+    type__motion_channels: required
```

Comment:
CI is complaining that some columns are declared here but do not exist in `[src/schema/objects/columns.yaml](https://github.com/bids-standard/bids-specification/pull/981/files#diff-f8514713cf38b0a053acb03265ef1b1b22245d6eeaeedc5a0a6663f3523441d4)`

https://github.com/bids-standard/bids-specification/actions/runs/4127444754/jobs/7130662091#step:5:101

```
E           tabular_data.motion.motionChannels.columns == type__motion_channels
E           tabular_data.motion.motionChannels.columns == tracked_point__channels
E           tabular_data.motion.motionChannels.columns == units__motion
E           tabular_data.motion.motionChannels.columns == placement__motion
```

Reply:
temporary commented to pass validation

Reply:
I added the missing column objects, should work now

Reply:
ok 
resolving this